ramls/inventory-batch.raml

```diff
@@ -13,26 +13,20 @@ types:
   instancesBatchResponse: !include instances-batch-response.json
 
 /inventory/instances/batch:
-    displayName: Instances batch API
-      type:
-        collection:
-          schemaCollection: instances
-          exampleCollection: !include examples/instancesBatch_post.json
-          schemaItem: instances
-          exampleItem: !include examples/instancesBatch_post.json
-    post:
-      description: "Create collection of instances in one request"
-      body:
-        application/json:
-          type: instances
-      responses:
-        201:
-          description: "Instances are created"
-          body:
-            application/json:
-              type: instancesBatchResponse
-        500:
-          description: "Internal server error"
-          body:
-            application/json:
-              type: instancesBatchResponse
+  displayName: Instances batch API
+  post:
+    description: "Create collection of instances in one request"
+    body:
+      application/json:
+        type: instances
+    responses:
+      201:
```

Comment:
Does the API respond with 201 when there are errors?

Reply:
No, this API responds with 500 Internal server error if there is any error on backend. 
I added tests for this case, see  InstancesApiExamples -> shouldReturnBadRequestIfOneInstancePostedWithoutTitle

Reply:
A `bad request` HTTP response is a status code within the 400 range. A 500 suggests that the server has encountered a problem.

It seems that 500 is being used to report a mixed outcome of success and failure. Improving this might not be in the scope of this work. If so, please change the name of the test(s) which check this, to something like `shouldReturnServerError...`

Reply:
Good suggestion, I will do this.

Reply:
Thanks